tests/paragraph.spec.ts

```diff
@@ -310,3 +310,23 @@ test('The key "Up" respond when the cursor located in first paragraph  ', async
   await page.keyboard.press('ArrowUp', { delay: 50 });
   await assertPageTitleFocus(page);
 });
+
+
+test('After deleting a row of the text, the cursor will jump to the end of the previous row of the list', async ({ page }) => {
```

Comment:
You should use 'should' instead of 'will' in unit tests to indicate it is expected behavior.

Reply:
Good catch!